source/standards/reliability-engineering.html.md.erb

```diff
@@ -0,0 +1,23 @@
+---
+title: Reliability Engineering
+expires: 2018-12-01
+---
+
+# <%= current_page.data.title %>
+
+Reliability Engineering provide a shared platform to GDS teams made up of tools and services to set up and maintain a service, for example by:
+
+* supporting the Amazon Web Services (AWS) and the GOV.UK PaaS infrastructure
+*  using [Logit][] to provision, manage and ensure availability of team's logging infrastructure
+*  running an operational metrics service using [Prometheus][]
+*  managing a shared base AWS account for GDS teams
```

Comment:
Following my last comment about the lead-in line I'd recommend tweaking the bullets a little to something like:

> - Amazon Web Services (AWS) and the GOV.UK PaaS infrastructure
> - Logit, for storing and querying infrastructure and application logs
> - Prometheus, for running an operational metrics service

Note that I removed the last bullet. Should AWS be mentioned twice given that the context is quite general?